invenio_theme_tugraz/config.py

```diff
@@ -25,6 +25,9 @@
 THEME_LOGO = 'images/tug_logo.png'
 """TU Graz logo"""
 
+ACCOUNTS_HEADER_LOGO =  'images/login_logo.png'
```

Comment:
```suggestion
ACCOUNTS_HEADER_LOGO = 'images/login_logo.png'
```
The extra space is failing the travis-CI